### ISSUE TITLE
msvcrt.dll:malloc/free not implemented

### ISSUE THREAD

**Issue by jrmuizel** [taviso/loadlibrary#25]

**Comment by jrmuizel** [comment]
Should these just call into the system malloc/free?

**Comment by taviso** [comment]
Yes, that will work - I did have this implemented, but I removed it - I rewrote how the imports work, and my test implementation (mpengine.dll) started using `LocalAlloc` instead.

This is common enough that I should add it back.
